state/open.go

```diff
@@ -165,11 +168,25 @@ func newState(
 		}
 	}()
 
+	mongodb := session.DB(jujuDB)
+	sstxn := featureflag.Enabled(feature.MongoDbSSTXN)
+	if sstxn {
+		if !txn.SupportsServerSideTransactions(mongodb) {
+			logger.Warningf("user requested server-side transactions, but they are not supported\n"+
+				" consider using the '%s' feature flag", feature.MongoDbSnap)
+			sstxn = false
```

Comment:
You could add a `logger.Infof("using client-side transactions")` here for parity with the other info statements OR  alternatively convert the `else` block on L181 into a standalone `if !sstxn {...}` block so you hit it after you force sstxn to false on L177